stratum/hal/lib/tdi/es2k/es2k_hal_test.cc

```diff
@@ -201,20 +201,6 @@ TEST_F(Es2kHalTest, ColdbootSetupSuccessForSavedConfigs) {
 
   EXPECT_CALL(*switch_mock_, PushChassisConfig(EqualsProto(chassis_config)))
       .WillOnce(Return(::util::OkStatus()));
-  EXPECT_CALL(
```

Comment:
Does this need to be added to the warmboot routine?

Reply:
I don't know. I did the minimum necessary to address the issue, which is due to differences in the way we handle pipeline configuration files.

My guess is, probably not. This test was adapted from the one for the Barefoot platform. We assume it did everything necessary until proven otherwise. The coverage analysis for the test might answer the question.

Reply:
The purpose of this PR is to make a failing unit test succeed.